### DISCUSSION TITLE
Vite config file should have default alias resolvers.

### DISCUSSION THREAD

**Discussion by yogeshgalav** [laravel/framework#50333] [General]
By default vite.config.js should come with following line of code
or can be added to docs
```
resolve: {
        alias: {
            '@': path.resolve(__dirname, 'resources/js'),
            '~': path.resolve(__dirname, 'node_modules'),
            '^': path.resolve(__dirname, 'public'),
        },
    },
```
It will keep standardization, and dictates 3 things to developers,
1. all frontend js file should be in `resources/js` and must be accessed by `@`
2. `node_modules` will be at root and must be accessed by `~`
3. All public files and images should be accessed from root `public` folder and must be accessed by `^`

Without public resolver one gets many errors while migrating to new laravel version or frontend migration to vite.
It's better to keep things versatile and abstracted but Laravel must dictate how to ideally use frontend framework with it.

**Comment by oriceon** [comment]
Anyone?
